apps/builder/remix.config.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
     "node-fetch",
     "immerhin",
     "strip-indent",
+    "camelcase",
```

Comment:
strange, we used this package before in many cases, but never added to this config, just to check if needed

Reply:
Always was in CLI. Now comes from tailwind parser in runtime.